### DISCUSSION TITLE
Modal component

### DISCUSSION THREAD

**Discussion by jasha1974** [wireui/wireui#118] [Q&A] [answered]
Hi, i'm begginer in laravel/livewire. So, could you explain me how works modal component ? put some example ?

thanks a lot

**Comment by ph7jack** [comment, marked as answer]
Hello @jasha1974 
First, learn about the livewire properties
https://laravel-livewire.com/docs/2.x/properties


The modal are using the `wire:model` (a property in the component) to open and close
In the property is stored the modal state (opened, closed)
You can change this state from a property
```
<x-modal wire:model.defer="simpleModal">
    <x-card title="Consent Terms">
        <p class="text-gray-600">
            Lorem Ipsum...
        </p>
 
        <x-slot name="footer">
            <div class="flex justify-end gap-x-4">
                <x-button flat label="Cancel" x-on:click="close" />
                <x-button primary label="I Agree" />
            </div>
        </x-slot>
    </x-card>
</x-modal>
```
More examples and usage
https://livewire-wireui.com/docs/modal

**Reply by jasha1974** [reply]
![image](https://user-images.githubusercontent.com/38765749/139858230-ba22ed22-c746-4c7b-85a5-fe27c67a8d97.png)
thanks  a lot !
